web/digital-twin-vis-framework/example-cesium-vis/README.md

```diff
@@ -167,27 +13,54 @@ A small amount of sample data has been committed to demonstrate the power of the
 - **New York**:
   - Tiled 3D buildings, loaded from a remote CMCL server.
   - 2D river data from a WMS endpoint provided by Cornell University.
+  - Sample tile sets to demonstrate clipping planes.
   - No metadata or timeseries present in this data set.
-  - Styling based on distance from Castle Clinton
-  - Filtering to hide the Whitehall Ferry terminal
 
-It's worth noting that with this sample data, no stack is running so no support for dynamic metadata or timeseries is available. This is something that we plan to work on in future, no true generic solution exists for this so far.
+It's also worth noting that with this example visualisation, no stack or FeatureInfoAgent is running, hence these is no support for dynamic metadata or timeseries is available. This is something that we plan to work on in future.
```

Comment:
"hence there is" ... "timeseries available"

Reply:
Updated.